packages/components/src/image-size-control/index.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * External dependencies
+ */
+import { isEmpty, noop } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Component } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Button from '../button';
+import ButtonGroup from '../button-group';
+import SelectControl from '../select-control';
+import TextControl from '../text-control';
+
+class ImageSizeControl extends Component {
```

Comment:
We didn't figure out yet a good place for all these block specific components, I don't think "components" is the best place for it as it's more the design system, maybe in block-editor module. We could consider making it priivate as a start in the block-library package.

Reply:
Yeah, there was discussion here #17148 about where this component should live, but it didn't really come to a conclusion. `block-editor` works for me, so I can move that PR ahead which can then move this one out of draft status :)